js/greeting/greetings.js

```diff
@@ -1,12 +1,58 @@
+import {displayTime} from './../time.js';
+
+console.log(displayTime().hour);
```

Comment:
@zaynaib - Would be needing the console log for a future revisit?

Reply:
@RafaelDavisH  No I don't think I will be needing it.